src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -400,6 +425,191 @@ class spell_pri_divine_hymn : public SpellScript
     }
 };
 
+Optional<ObjectGuid> GetDivineImageGUID(Unit* caster)
```

Comment:
This should be in Unit.cpp instead for general use, there are more PRs using a generic one GetSummonGUID(uint32 spellId)

Reply:
Change that one to Unit* because you're converting a Unit to a GUID just to get the Unit pointer via ObjectAccessor again, that doesnt make sense